subscription/downgrade.md

```diff
@@ -11,64 +11,32 @@ You can downgrade your Docker subscription anytime. When you downgrade your subs
 
 For example, if you are currently on a **Docker Team** subscription which is billed on the 8th of every month and you choose to downgrade to **Docker Personal** on the 15th, your plan will be active until 7th of the following month. You will be transferred to Docker Personal on the 8th of the following month.
 
-The following sections contain instructions on how to downgrade from a paid subscription to a Docker Personal tier.
+The following sections contain instructions on how to downgrade from a paid subscription to a Docker Personal subscription.
```

Comment:
As the aim is to cover all downgrade scenarios - Pro to Personal and Team to Free team, suggest we keep it generic. Something on the lines of "...instructions on how to downgrade your existing Docker subscription."

nit:  There's just one section now.